grades/models.py

```diff
@@ -40,16 +40,20 @@ def course_level(self):
     def __unicode__(self):
         return self.code
     
-    @permalink
+    def __str__(self):
+        return self.code
+
+    # FIX Permalink depercated
```

Comment:
Do you know what was the actual use for this?

Reply:
Not really, only that it was used for reverse the URL of courses